src/hotspot/share/prims/jvmti.xml

```diff
@@ -12870,8 +12870,8 @@ myInit() {
       reach this catch clause. If there are native methods on the call stack
       between the throw location and the catch clause, the exception may
       be reset by one of those native methods.
-      Similarly, exceptions that are reported as uncaught (<code>catch_klass</code>
-      et al. set to 0) may in fact be caught by native code.
+      Similarly, exceptions that are reported as uncaught (<code>catch_method</code>
+      and <code>catch_location</code> set to 0) may in fact be caught by native code.
```

Comment:
catch_method is a jmethodID so null if uncaught.

Reply:
Then you also need to fix:

"If there is no such catch clause, each field is set to 0."

Also, technically speaking, can't `catch_location` be 0 even if caught (caught first the bytecode of the method)? Although I doubt  javac would ever generate such code, it seems it is allowed. If so, then `catch_method == null` is the only check  a user should make.

Reply:
Thank you for the comments!
Yes, I've also come to the same conclusion about the only `catch_method == null` check.

> Then you also need to fix:
> "If there is no such catch clause, each field is set to 0."

Good catch, thanks.